### PR TITLE
Updated data cleaning notebook

### PR DESCRIPTION
**Pull request by zacharymeurer**
# Edits
Integrated anomaly detection process (isolation forests) for citizen data into data cleaning notebook. Additionally, centralized citizen and reference data cleaning into the data cleaning notebook. Then, cleaned up and commented a lot of the citizen data cleaning code and a little bit of the reference data cleaning code.

### Data cleaning notebook overview
input: alldata.csv & CSV files within pvttables_raw
output: CSV files within all data/citizen & all data/reference

### CODE REVIEW

**Review by AnaSof0 — APPROVED** [review]

File: data cleaning.ipynb [hunk 1]
```diff
@@ -2,7 +2,7 @@
  "cells": [
```
Comment (jacobe90) [inline data cleaning.ipynb:2]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Fill any missing state names in dataset
</pre><p>To be extra safe, we could consider running this on all datapoints</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (jacobe90) [inline data cleaning.ipynb:2]: <p>Nice job! The expression worked out for me when I plugged in 366 as the day of the year </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (jacobe90) [inline data cleaning.ipynb:2]: <p>Nice work! I would be interested to know how many datapoints this ended up removing?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>

Comment (jacobe90) [inline data cleaning.ipynb:2]: <p>Nice, thanks for merging reference and citizen data cleaning!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/BU-Spark/pitne-season-watch/pull/18/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='21'/>

**Review by colettebas — APPROVED** [review]
Looks good to me! I agree with the comments that Jacob left as after those are resolved, you're good to merge!

**Pull request closed without merging**